echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/PipelineConfigsPollingAgent.groovy

```diff
@@ -46,6 +46,9 @@ class PipelineConfigsPollingAgent extends AbstractPollingAgent {
   private final long intervalMs
   private final ActionsOperator actionsOperator
 
+  @Value('${scheduler.cron.timezone:America/Los_Angeles}')
```

Comment:
Could you add a constructor arg and put the `@Value` annotation there, and make the field final